config.json

```diff
@@ -20,6 +23,12 @@
             "Class" : "DNWS.DelayPlugin",
             "Preprocessing" : "false",
             "Postprocessing" :"false" 
+        },
+		{
+            "Path" : "clientinfo",
+            "Class" : "DNWS.ClientInfoPlugin",
```

Comment:
I don't think your code is correct. You name the plugin class name as "InfoPlugin", but here you set the parameter "Class" as ClientInfoPlugin, they are different.